### ISSUE TITLE
Run a final test of all package pages

### ISSUE THREAD

**Issue by daveverwer** [SwiftPackageIndex/SwiftPackageIndex-Server#212]
Brute force test, not only to check for broken packages but for edge case bugs.

I'll generate URLs for every Package page and open them 100 at a time in a browser 😬

Marked as blocked until we get a little closer to release. I don't want to do this twice

**Comment by finestructure** [comment]
You want to do this manually? 😅

**Comment by daveverwer** [comment]
Yep, just once. Shouldn't take very long.

**Comment by finestructure** [comment]
BTW, I've got a script that can [create a restfile from a sitemap](https://finestructure.co/blog/2019/8/2/website-testing-creating-a-restfile-from-sitemapxml?rq=sitemap) - for link testing.

Later on I was planning to generate a restfile to test links for popular packages in order to catch errors early as a smoke test. This could easily be extended to encompass the whole set tbh.

For my maintenance project I've scheduled an hourly run on Gitlab to run our API and website smoke tests, i.e. a home grown pingdom.
